lua/astrocommunity/pack/zig/README.md

```diff
@@ -6,9 +6,11 @@
 - curl / wget (required by zig-tools)
 - Neovim (>= 0.7.x)
 - ripgrep (>= 11.0)
+- Zig (>= 0.12.0)
```

Comment:
```suggestion
- Zig (>= 0.12.0 required for Neotest support)
```

Specify that the version is only required for the Neotest integration

Reply:
Oh, sorry, thank you for suggestion